replacy/util.py

```diff
@@ -7,17 +7,17 @@
 
 from replacy.db import get_match_dict_schema
 
-
 # set known extensions:
-known_string_extensions = ["description", "match_name", "category", "comment"]
+known_string_extensions = ["description", "match_name", "category", "subcategory", "comment", "suggestions_separator",
+                           "construct_suggestion_function"]
```

Comment:
is there any reason to do this? if they are in the match_dict, they will dynamically be added

Reply:
not every NLP service has `construct_suggestion_function` and `suggestions_separator`, so this is needed in order for `replacy/__init__.py ` code to work
anyway if we ended up changing `replacy/__init__.py ` this can be removed